tests/ert/unit_tests/test_run_path_creation.py

```diff
@@ -43,8 +42,10 @@ def func(ert_config):
             experiment_id, name="prior", ensemble_size=1
         )
         sample_prior(prior_ensemble, [0])
+        runargs = run_args(ert_config, prior_ensemble, 1)
+        runpaths = run_paths(ert_config)
         create_run_path(
-            run_args=run_args(ert_config, prior_ensemble, 1),
+            run_args=runargs,
```

Comment:
can we write "`run_args=run_args`" for underscore-consistency?

Reply:
No, it shadows the parameter.